dev/bots/pubspec.yaml

```diff
@@ -16,8 +16,9 @@ dependencies:
   process: 4.2.3
   test: 1.17.11
 
-  _fe_analyzer_shared: 23.0.0 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
-  analyzer: 2.0.0 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
+  _discoveryapis_commons: 1.0.1 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
```

Comment:
it looks like these additional deps were actually added to dev/devicelab in https://github.com/flutter/flutter/pull/88447/files